Advanced-Recruit-Training/src/BatteryStateOfChargeService.cpp

```diff
@@ -1,13 +1,24 @@
 #include "BatteryStateOfChargeService.h"
+#include <QDebug>
```

Comment:
Typically, we don't want to leave this in release code, especially if we don't use it.

Reply:
Same with the other includes